### ISSUE TITLE
Release ibis.iSDM 0.0.2

### ISSUE THREAD

**Issue by Martin-Jung** [iiasa/ibis.iSDM#38]
First release:

* [ ] `usethis::use_cran_comments()`
* [ ] Update (aspirational) install instructions in README
* [ ] Proofread `Title:` and `Description:`
* [ ] Check that all exported functions have `@return` and `@examples`
* [ ] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [ ] Check [licensing of included files](https://r-pkgs.org/license.html#code-you-bundle)
* [ ] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [ ] `git pull`
* [ ] `devtools::build_readme()`
* [ ] `urlchecker::url_check()`
* [ ] `devtools::check(remote = TRUE, manual = TRUE)`
* [ ] `devtools::check_win_devel()`
* [ ] `rhub::check_for_cran()`
* [ ] `git push`

Submit to CRAN:

* [ ] `usethis::use_version('patch')`
* [ ] `devtools::submit_cran()`
* [ ] Approve email

Wait for CRAN...

* [ ] Accepted :tada:
* [ ] `git push`
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version()`
* [ ] `git push`

**Comment by mhesselbarth** [comment]
Let me know if you need any help with this or have questions.

P.S. You dont need to run `devtools::build_readme()` anymore. Should be up-to-date due to the gh-action.

**Comment by Martin-Jung** [comment]
> Let me know if you need any help with this or have questions.
> 
> P.S. You dont need to run `devtools::build_readme()` anymore. Should be up-to-date due to the gh-action.

Ohh, any help appreciated. I tried to fix a few things, but the MacOS R CMD check is still failing atm.
First I think we want version 0.0.2 to the main branch, but need to have the R checks working

**Comment by mhesselbarth** [comment]
Aren't all checks failing at the moment? 

Okay, so the general roadmap would be fixing everything on `dev` (v0.0.2) first and then merging into `master`?

**Comment by mhesselbarth** [comment]
I lost a little bit track on what's still to do. Did you say you want to fix the remaining warnings about the documentation ?

**Comment by Martin-Jung** [comment]
Yes, the main outstanding things are all the documention warnings durings checks. I tried to fix some every now or then, but it is quite a lot...
